src/routes/index.js

```diff
@@ -7,5 +7,6 @@ const app = express.Router()
 
 app.use('/users', users)
 app.use('/articles', posts)
+app.use('/feed', posts)
```

Comment:
can you clarify the difference between a feed and an article?

Reply:
so the name of the endpoint according to the pdf is feed and not article, i am guessing a feed is a group of articles. I have renamed all the variables for clarity

Reply:
:slightly_smiling_face:  like i said earlier, my questions are just questions, it shouldn't trigger a change before you respond

Reply:
else you'll do more work than you're supposed to

Reply:
okay, also after taking my time to understand the question i realised i answered wrongly.

So according to the pdf where the assignment is spelt in, the `articles` endpoint is for creating articles
The `feed` endpoint is for getting articles

Reply:
okay cool, thanks for the clarification